pkg/manager/member/executor_manager.go

```diff
@@ -725,7 +718,158 @@ func (m *executorMemberManager) getContainerByName(sts *appsv1.StatefulSet, cont
 	return nil
 }
 
-func (m *executorMemberManager) getVolsByName(tc *v1alpha1.TiflowCluster, sts *appsv1.StatefulSet, podName string) (map[string]*v1alpha1.StorageVolumeStatus, error) {
+func (m *executorMemberManager) syncVolsStatus(tc *v1alpha1.TiflowCluster, sts *appsv1.StatefulSet) error {
 	// todo:
-	return nil, nil
+	selector, err := metav1.LabelSelectorAsSelector(sts.Spec.Selector)
+	if err != nil {
+		klog.Errorf("converting statefulSet %s selector to metav1 selector", sts.Name)
+		return err
+	}
+
+	pvs, err := m.clientSet.CoreV1().PersistentVolumes().List(context.TODO(), metav1.ListOptions{
+		LabelSelector: selector.String(),
+	})
+
+	for _, pv := range pvs.Items {
+		tc.Status.Executor.Volumes[pv.Name] = &v1alpha1.StorageVolumeStatus{
+			Name: DefaultStorageName,
+			ObservedStorageVolumeStatus: v1alpha1.ObservedStorageVolumeStatus{
+				BoundCount:      1,
```

Comment:
why hardcode here

Reply:
no use, will develop this later

Reply:
Later, we need to calculate the current usage information of each executor's pvc and pv in real time